src/workloads/scale/LargeScaleLongLived.yml

```diff
@@ -90,7 +90,9 @@ AutoRun:
       - atlas
       - replica
       - replica-all-feature-flags
+      - shard
       - single-replica
       - single-replica-15gbwtcache
-      - shard
       - standalone
+      - standalone-classic-query-engine
```

Comment:
I was looking at your sys-perf patch build to confirm that this BV was running LargeScaleLongLived, and saw that it didn't run the standalone classic variant. I re-configured the patch build to do so, and will double check that this work (for future reference, is it ok to reconfigure someone else's patch build?)